.pre-commit-config.yaml

```diff
@@ -2,22 +2,22 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+    rev: v3.4.0
     hooks:
       - id: trailing-whitespace
       - id: check-yaml
       - id: check-added-large-files
   - repo: https://github.com/PyCQA/isort
-    rev: 5.5.1
+    rev: 5.8.0
     hooks:
       - id: isort
   - repo: https://github.com/humitos/mirrors-autoflake.git
-    rev: v1.3
+    rev: v1.1
```

Comment:
this is weird why updating pre-commit would revert the version of autoflake to an older version?

Reply:
not sure why. it had me update pre-commit before checks could be run. doesn't seem to be an issue for now